python/apps/taiga/src/taiga/stories/attachments/events/__init__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2023-present Kaleidos INC
+
+from typing import cast
+
+from taiga.attachments.models import Attachment
+from taiga.events import events_manager
+from taiga.projects.projects.models import Project
+from taiga.stories.attachments.events.content import CreateStoryAttachmentContent
+from taiga.stories.stories.models import Story
+
+CREATE_STORY_ATTACHMENT = "stories.attachments.create"
+
+
+async def emit_event_when_story_attachment_is_created(
```

Comment:
missing documentation?

Reply:
Nope, https://github.com/taigaio/taiga/pull/442/files#diff-fabd36df6afc004f34c59ef0336109cc035435d7bf41d02f546d455c49385589

Reply:
oops! then I missed it. checking!